api/shared/nodenetworkconfigurationpolicy_types.go

```diff
@@ -12,18 +12,21 @@ type NodeNetworkConfigurationPolicySpec struct {
 	// The desired configuration of the policy
 	DesiredState State `json:"desiredState,omitempty"`
 
-	// When set to true, changes are applied to all nodes in parallel
+	// Percentage of matching cluster nodes that may apply policy configuration concurrently.
+	// If not specified, policy is applied one node at a time.
+	// Value of 0 means all nodes apply policy in parallel,
```

Comment:
Sinze `n` is percentage the full parallel has to be n = 100, it will means that the ChunkSize is 100% of the cluster.